jobs/tests.py

```diff
@@ -1,3 +1,28 @@
 from django.test import TestCase
 
-# Create your tests here.
+from model_mommy import mommy
+
+from jobs.models import Job
+
+
+class JobMethodTests(TestCase):
```

Comment:
Suggest "JobModelTests" (since later we will have View tests and Form tests etc, which may also have methods)